saleor/graphql/webhook/mutations.py

```diff
@@ -306,3 +321,96 @@ def perform_mutation(cls, _root, info: ResolveInfo, **data):
         manager = get_plugin_manager_promise(info.context).get()
         manager.event_delivery_retry(delivery)
         return EventDeliveryRetry(delivery=delivery)
+
+
+class WebhookDryRun(BaseMutation):
```

Comment:
BTW we try to use the pattern that every mutation is a separate file, once we merge dry-run and manual-trigger, we could quickly refactor this file. Just a note.

Reply:
Will be handled in separate PR